Services/AzineMailgunTwigExtension.php

```diff
@@ -18,6 +32,18 @@ public function getFilters()
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getGlobals()
+    {
+        $lastKnownIp = is_null($this->repository->getLastKnownSenderIp()) ? '' : $this->repository->getLastKnownSenderIp();
```

Comment:
Adding this here will cause this code to be executed every time **any** page rendered on the site. => bad idea. :-(